tools/package-tools/build-bundles.ts

```diff
@@ -1,57 +1,101 @@
 import {join} from 'path';
+import {readdirSync, lstatSync} from 'fs';
 import {ScriptTarget, ModuleKind, NewLineKind} from 'typescript';
 import {uglifyJsFile} from './minify-sources';
 import {createRollupBundle} from './rollup-helpers';
 import {remapSourcemap} from './sourcemap-remap';
 import {transpileFile} from './typescript-transpile';
 import {buildConfig} from './build-config';
+import {getSecondaryEntryPointsForPackage} from './secondary-entry-points';
 
 /** Directory where all bundles will be created in. */
 const bundlesDir = join(buildConfig.outputDir, 'bundles');
 
-/** Builds the bundles for the specified package. */
-export async function buildPackageBundles(entryFile: string, packageName: string) {
-  const moduleName = `ng.${packageName}`;
 
-  // List of paths to the package bundles.
-  const fesm2015File = join(bundlesDir, `${packageName}.js`);
-  const fesm2014File = join(bundlesDir, `${packageName}.es5.js`);
-  const umdFile = join(bundlesDir, `${packageName}.umd.js`);
-  const umdMinFile = join(bundlesDir, `${packageName}.umd.min.js`);
+/** Builds bundles for the primary entry-point w/ given entry file, e.g. @angular/cdk */
+export async function buildPrimaryEntryPointBundles(entryFile: string, packageName: string) {
+  return createBunldesForEntryPoint({
+    entryFile,
+    moduleName:   `ng.${packageName}`,
+    fesm2015Dest: join(bundlesDir, `${packageName}.js`),
+    fesm2014Dest: join(bundlesDir, `${packageName}.es5.js`),
+    umdDest:      join(bundlesDir, `${packageName}.umd.js`),
+    umdMinDest:   join(bundlesDir, `${packageName}.umd.min.js`),
```

Comment:
Nit (optional): I kinda hate indenting things using spaces like that. Up to preference though :)